packages/pigment-css-unplugin/src/index.ts

```diff
@@ -357,9 +357,9 @@ export const plugin = createUnplugin<PigmentOptions, true>((options) => {
           map: result.sourceMap,
         };
       } catch (e) {
-        const error = new Error((e as Error).message);
-        error.stack = (e as Error).stack;
-        throw error;
+        // const error = new Error((e as Error).message);
+        // error.stack = (e as Error).stack;
+        // throw error;
```

Comment:
I think the `e.message` is needed further down the chain to get dispalyed in webpacks Error view. But have not figured out a better way of handling this. Maybe some help/pointers here?